administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue

```diff
@@ -9,6 +9,7 @@
             </a>
         </div>
         <media-breadcrumb></media-breadcrumb>
+        <input type="text" @input="changeSearch" placeholder="Search item.."/>
```

Comment:
The placeholder needs to be a language string. And please use 3 instead of 2 dots.